### PR TITLE
{tools}[dummy/] wkhtmltopdf v0.12.3

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a33d204866c28db2cc370a89222d1b31 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by pescobar** [comment]
this one has many system deps: http://wkhtmltopdf.org/downloads.html

`depends on: zlib, fontconfig, freetype, X11 libs (libX11, libXext, libXrender)`

**Comment by boegel** [comment]
@pescobar should we list them in `osdependencies`? I wouldn't go and include them in real deps for a dummy install...

**Comment by pescobar** [comment]
@boegel I added os deps for redhat/ubuntu

File: easybuild/easyconfigs/w/wkhtmltopdf/wkhtmltopdf-0.12.3-Linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description = """wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render 
+ HTML into PDF and various image formats using the Qt WebKit rendering engine. These run entirely headless 
+ and do not require a display or display service."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://download.gna.org/wkhtmltopdf/%(version_major_minor)s/%(version)s/']
+sources = ['wkhtmltox-%(version)s_linux-generic-amd64.tar.xz']
+
+osdependencies = [('zlib', 'zlib1g'), 'fontconfig', ('freetype', 'libfreetype6'), ('libX11', 'libx11-6'), ('libXext', 'libxext6'), ('libXrender', 'libxrender1')]
```
Comment (boegel) [inline easybuild/easyconfigs/w/wkhtmltopdf/wkhtmltopdf-0.12.3-Linux-x86_64.eb]: @pescobar line too long, let's go with one per line?

```python
osdependencies = [
    ('zlib', 'zlib1g'),
    'fontconfig',
    ('freetype', 'libfreetype6'),
    ('libX11', 'libx11-6'),
    ('libXext', 'libxext6'),
    ('libXrender', 'libxrender1'),
]
```

**Comment by boegel** [comment]
@pescobar no need to close/open on every push you do, Travis is picking up new commits correctly now?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/709d50030a74635dfe0981f70e072523 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**